crates/lunatic-distributed/src/control/server.rs

```diff
@@ -79,15 +84,59 @@ impl Server {
     }
 }
 
-impl Default for Server {
-    fn default() -> Self {
-        Self::new()
+pub static CTRL_SERVER_NAME: &'static str = "ctrl.lunatic.cloud";
+pub static ROOT_CERT: &'static str = include_str!("../../certs/root.pem");
+static ROOT_KEYS: &'static str = include_str!("../../certs/root.keys.pem");
+
+pub fn root_cert(
+    test_ca: bool,
+    ca_cert: Option<&str>,
+    ca_keys: Option<&str>,
+) -> Result<Certificate> {
+    if test_ca {
+        let key_pair = KeyPair::from_pem(ROOT_KEYS)?;
+        let root_params = CertificateParams::from_ca_cert_pem(ROOT_CERT, key_pair)?;
+        let root_cert = Certificate::from_params(root_params)?;
+        Ok(root_cert)
+    } else {
+        let ca_cert_pem = std::fs::read(Path::new(ca_cert.unwrap()))?;
```

Comment:
Unwrap on option... use `ok_or_else` and `?` https://doc.rust-lang.org/std/option/enum.Option.html#method.ok_or_else